templates/magazine/panel/moderator_requests.html.twig

```diff
@@ -54,7 +54,7 @@
                                     <button type="submit"
                                             class="btn btn__secondary"
                                             title="{{ 'reject'|trans }}">
-                                        <i class="fa-solid fa-ban"></i>
+                                        <i class="fa-solid fa-ban"></i> {{ 'reject'|trans }}
```

Comment:
Is the `{{ 'reject'|trans }}` doing something on this line?

Reply:
it adds the label after the button to make it clearer what it's for.

oh, whoops, it should also be in a `<span>`; my bad, one second

before
![image](https://github.com/MbinOrg/mbin/assets/146029455/5ba60bab-2b57-45d3-9db0-79fc39ada222)
after
![image](https://github.com/MbinOrg/mbin/assets/146029455/f5742f41-b29d-49c6-9ea7-f67c589cdf6e)